main.nf

```diff
@@ -2127,6 +2133,33 @@ process mapdamage_rescaling {
 
 // Optionally perform further aDNA evaluation or filtering for just reads with damage etc.
 
+process mask_reference_for_pmdtools {
+    label 'sc_tiny'
+    tag "${fasta}"
+    publishDir "${params.outdir}/reference_genome/masked_reference", mode: params.publish_dir_mode
+
+    when: (params.pmdtools_reference_mask && params.run_pmdtools)
+
+    input:
+    file fasta from ch_unmasked_fasta_for_masking.collect()
```

Comment:
Do you really need `collect` here? You would only need to run this process once, right?

Reply:
Must have been leftover from copy-pasting the fasta input of the `pmdtools` process. Will remove.